doc/tutorials/modeling-threat-intel-ctim.md

```diff
@@ -405,9 +405,9 @@ A Verdict indicates the *most recent* and *most relevant* disposition for a give
 
 ### 1.5: Sighting Entities
 
-A **Sighting** is a record of the appearance of a cyber observable at a given date and time.
+A **Sighting** is a record of the appearance of a cyber threat Indicator match at a given date and time.  This can be a pattern match in a rule matching engine or expert system, or an observable feed based indicator such as an IP or domain blacklist.
```

Comment:
The Sighting defined in the [STIX 2 specification](http://docs.oasis-open.org/cti/stix/v2.0/cs01/part2-stix-objects/stix-v2.0-cs01-part2-stix-objects.html#_Toc496714342) is not limited to the appearance of an Indicator.

> A Sighting denotes the belief that something in CTI (e.g., an indicator, malware, tool, threat actor, etc.) was seen.

Is the CTIM Sighting different?

Reply:
Yes. Sightings in CTIM do not relate directly to actors, tools, malware, attack patterns, vulnerabilities, etc. They relate to indicators.  Indicators relate to those other things.